### DISCUSSION TITLE
v1.8.2版本docker-compose部署遇到的问题

### DISCUSSION THREAD

**Discussion by gnalli** [hhyo/Archery#1420] [General]
v1.8.2版本使用docker-compose部署的时候，docker-compose.yaml文件里面使用的镜像hhyo/archery:1.8.2 tag号应该是v1.8.2，不然拉取镜像会报错

**Comment by gnalli** [comment]
这个压缩包：v1.8.2.tar.gz

**Reply by LeoQuote** [reply]
master 分支已经修了
